oak-run-commons/src/main/java/org/apache/jackrabbit/oak/fixture/DataStoreUtils.java

```diff
@@ -146,26 +150,52 @@ public static void deleteBucket(String bucket, Map<String, ?> map, Date date) th
     }
 
     public static void deleteAzureContainer(Map<String, ?> config, String containerName) throws Exception {
+        if (config == null) {
+            return;
+        }
         if (Strings.isNullOrEmpty(containerName)) {
             return;
         }
-        String connectionString = (String) config.get(AzureConstants.AZURE_CONNECTION_STRING);
-        if (Strings.isNullOrEmpty(connectionString)) {
-            String accountName = (String) config.get(AzureConstants.AZURE_STORAGE_ACCOUNT_NAME);
-            String accountKey = (String) config.get(AzureConstants.AZURE_STORAGE_ACCOUNT_KEY);
-            if (Strings.isNullOrEmpty(accountName) ||
-                Strings.isNullOrEmpty(accountKey)) {
-                return;
-            }
-            connectionString = org.apache.jackrabbit.oak.blob.cloud.azure.blobstorage.Utils.getConnectionString(accountName, accountKey);
+        CloudBlobContainer container = getCloudBlobContainer(config, containerName);
+        if (container == null) {
+            log.info("container is not initialized");
+            return;
         }
         log.info("deleting container [{}]", containerName);
-        CloudBlobContainer container = 
-            org.apache.jackrabbit.oak.blob.cloud.azure.blobstorage.Utils.getBlobContainer(connectionString, containerName);
         if (container.deleteIfExists()) {
             log.info("container [{}] deleted", containerName);
         } else {
             log.info("container [{}] doesn't exists", containerName);
         }
     }
-}
+
+    @Nullable
+    private static CloudBlobContainer getCloudBlobContainer(@NotNull Map<String, ?> config,
+                                                            @NotNull String containerName) throws DataStoreException {
+        final String azureConnectionString = (String) config.get(AzureConstants.AZURE_CONNECTION_STRING);
+        final String clientId = (String) config.get(AzureConstants.AZURE_CLIENT_ID);
+        final String clientSecret = (String) config.get(AzureConstants.AZURE_CLIENT_SECRET);
+        final String tenantId = (String) config.get(AzureConstants.AZURE_TENANT_ID);
+        final String accountName = (String) config.get(AzureConstants.AZURE_STORAGE_ACCOUNT_NAME);
+        final String accountKey = (String) config.get(AzureConstants.AZURE_STORAGE_ACCOUNT_KEY);
+        final String blobEndpoint = (String) config.get(AzureConstants.AZURE_BLOB_ENDPOINT);
+        final String sasToken = (String) config.get(AzureConstants.AZURE_SAS);
+
+        if (StringUtils.isAllBlank(azureConnectionString, clientId, clientSecret, tenantId, accountName, accountKey)) {
+            return null;
```

Comment:
log warn statement as to why the container is null

Reply:
done